writers/cassandra/README.md

```diff
@@ -18,7 +18,7 @@ default values.
 | MF_CASSANDRA_WRITER_DB_USER         | Cassandra DB username                                     |                        |
 | MF_CASSANDRA_WRITER_DB_PASS         | Cassandra DB password                                     |                        |
 | MF_CASSANDRA_WRITER_DB_PORT         | Cassandra DB port                                         | 9042                   |
-| MF_CASSANDRA_WRITER_CONFIG_PATH     | Configuration file path with filters list                 | /configs.toml          |
+| MF_CASSANDRA_WRITER_SUBJECTS_CONFIG | Configuration file path with subjects list                | /config/subjects.toml  |
```

Comment:
I would only add here maybe `NATS subjects list`, or the people who land in the project and do not know NATS terminology.

Reply:
This could be `config.toml` and contain 
```
[nats]
subjects = ["channels.>"]

[transformer]
filter=["temperature", "humidity"]
```

Reply:
I kept `MF_<NAME>_WRITER_CONFIG_PATH` and updated code, docs, and compose files to match it. In README files, we mention the NATS subject, so it should be clear. I removed the `keys` filter, but we'll possibly add something like that later - it's a nice option, but I'm not sure if it belongs to the Writers or Transformers. That's also the reason I didn't want to use NATS in config file path naming - it will probably be used for more complex configuration in the future and some of that config won't be related to NATS. This will probably need an update in k8s scripts (but nothing is broken due to the sensible defaults).